rotkehlchen/chain/ethereum/modules/airdrops/decoder.py

```diff
@@ -41,13 +49,32 @@
 FOX_CLAIMED = b"R\x897\xb30\x08-\x89*\x98\xd4\xe4(\xab-\xcc\xa7\x84KQ\xd2'\xa1\xc0\xaeg\xf0\xb5&\x1a\xcb\xd9"  # noqa: E501
 
 
+ELFI_LOCKING = string_to_ethereum_address('0x02Bd4A3b1b95b01F2Aa61655415A5d3EAAcaafdD')
+ELFI_VOTE_CHANGE = b'3\x16\x1c\xf2\xda(\xd7G\xbe\x9d\xf16\xb6\xf3r\x93\x90)\x84\x94\x94rht1\x93\xc5=s\xd3\xc2\xe0'  # noqa: E501
+# ELFI_AIRDROP1 = string_to_ethereum_address('0x5ae69B714859A3C15281e0a227D9B8C82F03b966')
+# ELFI_AIRDROP2 = string_to_ethereum_address('0x72D3acDAd21dF959DB2C112A0a5982d03759a154')
```

Comment:
This commented addresses can be removed?

Reply:
ah yes yes